javascript/README.md

```diff
@@ -91,6 +91,8 @@ Code snippets in Javascript demonstrating how to use various modules of the [Med
   * [tag.js](tag.js): Remove the spam tag from log entry ID 123 with the reason Wrongly applied
 * [API:Duplicatefiles](https://www.mediawiki.org/wiki/API:Duplicatefiles)
   * [duplicate_files.js](duplicate_files.js): list duplicates of the given files
+* [API:Iwbacklinks](https://www.mediawiki.org/wiki/API:Iwbacklinks)
+  * [iwbacklinks.js](iwbacklinks.js): Get pages linking to wikibooks:Main_Page.
```

Comment:
Let's make this `Get pages that link to a given interwiki link`

Reply:
I've fixed this